Source/Core/Core/WiiRoot.cpp

```diff
@@ -26,12 +28,10 @@ namespace Core
 {
 static std::string s_temp_wii_root;
 
-static void InitializeDeterministicWiiSaves()
+static void InitializeDeterministicWiiSaves(IOS::HLE::FS::FileSystem* session_fs)
```

Comment:
I really liked the `FS` namespace shortcut on the other PR, might be worthwhile doing something here too to cut down on the boilerplate and focus more on what matters.

Reply:
Good idea, I've added an alias to reduce noise.